app.css

```diff
@@ -1,24 +1,50 @@
-/*.list--bare {
-  list-style: none;
+body {
   padding: 0;
   margin: 0;
+  font-family: Helvetica; Helvetica Nueva;
+  background-color: #d0d0d0;
 }
-.row {
-  font-size: 0;
+h1 {
+  text-align: center;
+  color: #000;
+  margin: 5% 5%;
 }
-.color {
-  display: inline-block;
-  font-size: 1rem;
-  width: 25%;
-  padding: 0 20px;
-  box-sizing: border-box;
-  margin-bottom: 40px;
+h3 {
+  color: #000;
+  background-color: #fff;
+  text-transform: capitalize;
+
+}
+.palette{
   text-align: center;
-  font-family: Helvetica;
-  color: #4D4D4D;
-}
-.color img {
-  width: 100%;
-  height: 200px;
-  display: block;
-}*/
+}
+.dominant {
+  display: inline-block;
+  margin: .5em;
+  height: 100px;
+  width: 150px;
+}
+.contrastingDominant {
+  display: inline-block;
+  margin: .5em;
+  height: 100px;
+  width: 100px;
+}
+.subDominant {
+  display: inline-block;
+  margin: .5em;
+  height: 100px;
+  width: 150px;
+}
+.contrastingSubDominant {
+  display: inline-block;
+  margin: .5em;
```

Comment:
I recommend prefixing this with a `0` so that it's clear to anyone that this is a decimal